### PR TITLE
Map & city analytics v1.0

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/LD-230
- Will need to revisit analytics once all city list views are enabled.
- I've marked a "y" next to rows that are finished in the spreadsheet, and all "green" rows are finished

#native_no_changes

### CODE REVIEW

File: src/lib/Scenes/Map/Components/ShowCard.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ interface ShowCardProps {
   shows: ShowItemRow_show[]
   onSaveStarted?: () => void
   onSaveEnded?: () => void
+  onSave?: () => void
```
Comment (l2succes) [inline src/lib/Scenes/Map/Components/ShowCard.tsx]: We don't need this anymore

File: src/lib/Scenes/Map/Components/ShowCard.tsx [hunk 2]
```diff
@@ -52,7 +53,7 @@ export class ShowCard extends Component<ShowCardProps, ShowCardState> {
     currentPage: 1,
     isSaving: false,
   }
-
+  Background
```
Comment (l2succes) [inline src/lib/Scenes/Map/Components/ShowCard.tsx]: ?

**Comment by l2succes** [comment]
merge on green

File: src/lib/Scenes/Map/Components/ShowCard.tsx (hunk 2 above)
Comment (orta) [reply src/lib/Scenes/Map/Components/ShowCard.tsx]: Addressed, and the other commetn

**Pull request merged**